src/vmm/src/lib.rs

```diff
@@ -1656,19 +1656,18 @@ mod tests {
         }};
     }
 
-    extern crate tempfile;
-
-    use super::*;
+    extern crate utils;
```

Comment:
```suggestion
```

And while you're at it please also remove:
```rust
extern crate logger;
extern crate utils;
```

in `src/vmm/src/signal_handler.rs`.